README.md

```diff
@@ -220,33 +200,32 @@ The predict endpoint returns a prediction response in JSON. It will look somethi
 ]
 ```
 
-You will see this result in the response to your `curl` call to the predict endpoint, and in the server logs in the terminal window running TorchServe. It's also being [logged locally with metrics](docs/metrics.md).
+All interactions with the endpoint will be logged in the `logs/` directory, so make sure to check it out!
 
 Now you've seen how easy it can be to serve a deep learning model with TorchServe! [Would you like to know more?](docs/server.md)
 
-### Stop the running TorchServe
+### Stop TorchServe
 
-To stop the currently running TorchServe instance, run the following command:
+To stop the currently running TorchServe instance, run:
 
 ```bash
 torchserve --stop
 ```
 
-You see output specifying that TorchServe has stopped.
-
-
 ### Concurrency And Number of Workers
-TorchServe exposes configurations that allow the user to configure the number of worker threads on CPU and GPUs. There is an important config property that can speed up the server depending on the workload.
-*Note: the following property has bigger impact under heavy workloads.*
-If TorchServe is hosted on a machine with GPUs, there is a config property called `number_of_gpu` that tells the server to use a specific number of GPUs per model. In cases where we register multiple models with the server, this will apply to all the models registered. If this is set to a low value (ex: 0 or 1), it will result in under-utilization of GPUs. On the contrary, setting to a high value (>= max GPUs available on the system) results in as many workers getting spawned per model. Clearly, this will result in unnecessary contention for GPUs and can result in sub-optimal scheduling of threads to GPU.
-```
-ValueToSet = (Number of Hardware GPUs) / (Number of Unique Models)
-```
-
+TorchServe exposes configurations that allow the user to configure the number of worker threads on CPU and GPUs. `number_of_gpu` determines the number of GPUs *per model* and should be set to `Number of GPUs / Number of Unique Models` to avoid under-utilization and GPU contention
 
 ## Quick Start with Docker
 Refer to [torchserve docker](docker/README.md) for details.
 
+## Highlighted Examples
+* [HuggingFace Transformers](examples/Huggingface_Transformers)
+* [MultiModal models with MMF](https://github.com/pytorch/serve/tree/master/examples/MMF-activity-recognition) combining text, audio and video
+* Complex workflows, models chained together in a dependency graph
+  - [Dog Breed Classification](examples/Workflows/dog_breed_classification)
+  - [Neural Machine Translation](examples/Workflows/nmt_tranformers_pipeline)
+
```

Comment:
Please add a link to the main examples readme as well, and mention for other examples please see....